recipes/sisppeo/meta.yaml

```diff
@@ -0,0 +1,75 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "SISPPEO" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/inrae/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: b9b11be0cf68642d9c7e608800889d2f11bce6358ced1184e351b7e38da2eeb8
+
+build:
+  number: 0
+  skip: True  # [osx or py<38]
```

Comment:
This package is pure Python. It should be a `noarch: python`
Remove `skip` and set a minimum version on Python requirement instead.

noarch packages are only built on Linux. If build fails on windows or osx in the staged-recipes (because some dependencies aren't available on that platform), it doesn't matter.